nilearn/image/image.py

```diff
@@ -502,6 +502,58 @@ def swap_img_hemispheres(img):
 
     # create swapped nifti object
     out_img = nibabel.Nifti1Image(img.get_data()[::-1], img.get_affine(),
-        header=img.get_header())
+                                  header=img.get_header())
 
     return out_img
+
+
+def _index_img(imgs, index):
+    """Helper function for index_img and iter_img
+    """
+    return nibabel.Nifti1Image(imgs.get_data()[..., index],
```

Comment:
> Do we trust that any user who knows the difference between these
> expressions also knows about memory-related issues of their use?

Clueless users don't, but we cannot solve their problems with low-level
code. Advanced users do, and we should enable them by no trying to
second-guess their goals.